vmdb/app/controllers/catalog_controller.rb

```diff
@@ -730,7 +730,7 @@ def ot_remove_submit
       else
         add_flash(_("Orchestration Template \"%s\" was deleted.") % ot.name)
       end
-      self.x_node = 'root'
+      self.x_node = ot.type == "OrchestrationTemplateHot" ? "xx-othot" : "xx-otcfn"
```

Comment:
prefer `ot.kind_of?(OrchestrationTemplateHot)` to `ot.type == "OrchestrationTemplateHot"`, if that is the same thing.